Compilation error due to merge in r0.0-alpha

**Problem description**
The merge introduced a conflict and the branch build is now failing with the following error:
```
:controller:server:compileJava/home/travis/build/pravega/pravega/controller/server/src/main/java/com/emc/pravega/controller/server/ControllerService.java:266: error: method runAsync in class RetryAndThrowBase<ThrowsT> cannot be applied to given types;
                .throwingOn(RuntimeException.class).runAsync(() -> streamTransactionMetadataTasks
                                                   ^
  required: Supplier<CompletableFuture<ReturnT>>,ScheduledExecutorService
  found: ()->stream[...]null),Executor
  reason: cannot infer type-variable(s) ReturnT
    (argument mismatch; Executor cannot be converted to ScheduledExecutorService)
  where ReturnT,ThrowsT are type-variables:
    ReturnT extends Object declared in method <ReturnT>runAsync(Supplier<CompletableFuture<ReturnT>>,ScheduledExecutorService)
    ThrowsT extends Exception declared in class RetryAndThrowBase
1 error
 FAILED
```

**Problem location**
Controller service.

--- a/controller/server/src/main/java/com/emc/pravega/controller/server/ControllerService.java
+++ b/controller/server/src/main/java/com/emc/pravega/controller/server/ControllerService.java
@@ -43,7 +43,6 @@ import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
 import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.Executor;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.stream.Collectors;
 import lombok.AllArgsConstructor;

